WalletWasabi.Fluent/ViewModels/Dialogs/EnterPasswordDialogViewModel.cs

```diff
@@ -1,50 +0,0 @@
-using System.Reactive.Linq;
```

Comment:
Why delete this? We will likely need password entry without the context of signing a transaction?

Reply:
Still possible with the `PasswordAuthDialogViewModel`